D/DiffEqProblemLibrary/Compat.toml

```diff
@@ -32,5 +32,11 @@ ModelingToolkit = "0.0.0 - 0.9"
 ["4.5.0"]
 DiffEqBase = "5.15.0 - 6"
 
-["4.5.1-4"]
-DiffEqBase = "5.15.0-*"
+["4.5.1-4.5"]
+DiffEqBase = "5.15.0- 6"
```

Comment:
@ChrisRackauckas Can you change line 36 to `DiffEqBase = "5.15.0 - 6"` to be consistent with the whitespace style of line 33?

Reply:
Did registrator write this?

Reply:
@ChrisRackauckas added this line manually

Reply:
Ah, didn't notice the extra commit. Automerge should probably not have approved this.